lib_term/dot.ml

```diff
@@ -30,7 +30,9 @@ let node f ?style ?shape ?bg ?url ?tooltip i label =
   let tooltip = Option.map (limit_str 4096) tooltip in (* (Graphviz max length is 16384) *)
   let attrs = [
     "label", Some label;
+    "color", bg;
     "fillcolor", bg;
+    "fontname", Some "Roboto, sans-serif";
```

Comment:
It might be better to set the font globally, rather than duplicate it on every node. e.g. https://stackoverflow.com/a/10880578/50926

Reply:
I have done this.  There is now a new changeset in this PR, which adds a helper for writing the digraph stanza.  The fontname is then added at that point.